docs/unique/board-automation-improvements.md

```
@@ -93,26 +93,26 @@ references:
 Automation of the board needs some serious work. A lot of these automaticly generated tasks were sort of hidden from me til I fixed the scripts enough for them to not totally ruin the board.
 <!-- GENERATED-SECTIONS:DO-NOT-EDIT-BELOW -->
 ## Related content
-- [Board Walk – 2025-08-11](board-walk-2025-08-11.md)
-- [Admin Dashboard for User Management](admin-dashboard-for-user-management.md)
-- [Dynamic Context Model for Web Components](dynamic-context-model-for-web-components.md)
-- [AI-Centric OS with MCP Layer](ai-centric-os-with-mcp-layer.md)
-- [AI-First-OS-Model-Context-Protocol](ai-first-os-model-context-protocol.md)
-- [balanced-bst](balanced-bst.md)
+- [[board-walk-2025-08-11|Board Walk – 2025-08-11]]
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)